app/src/ui/changes/sidebar.tsx

```diff
@@ -168,12 +168,16 @@ export class ChangesSidebar extends React.Component<IChangesSidebarProps, {}> {
   }
 
   private onDiscardAllChanges = (
-    files: ReadonlyArray<WorkingDirectoryFileChange>
+    files: ReadonlyArray<WorkingDirectoryFileChange>,
+    dontDisplayAsAll?: boolean
```

Comment:
what do you think about using a default value here instead of an optional? that way we can avoid checking for `undefined` in the ternary

Reply:
Sounds great! I definitely like that suggestion better.